src/components/panel/_macro-options.md

```diff
@@ -1,13 +1,13 @@
-| titleTag   | string  | false    | The html tag to wrap the title text in. Will default to a `div`                             |
-| type       | string  | false    | The type of panel to render. Available options are `success`, `warn`, `error` and `branded` |
-| spacious   | boolean | false    | Will render a more spacious version of the panel if set to `true`                           |
-| classes    | string  | false    | Custom classes to add to the panel                                                          |
-| id         | string  | false    | Custom id to add to the panel                                                               |
-| attributes | object  | false    | HTML attributes to apply to the panel (e.g. data attributes)                                |
-| icon       | string  | false    | Set this to the name of the icon you want to be included before the contents of the panel   |
-| iconSize   | string  | false    | Sets the size of the icon "s" for small and "l" for large. If not set will be defualt       |
-| iconsPath  | string  | false    | Set this to the path to the icon you want to be included before the contents of the panel   |
+| Name       | Type    | Required | Description                                                                                           |
+| ---------- | ------- | -------- | ----------------------------------------------------------------------------------------------------- |
+| body       | string  | true     | The contents of the panel. This can be a string of HTML                                               |
+| title      | string  | false    | The title for the panel. If this is not provided the inline/simple version will be rendered           |
+| titleTag   | string  | false    | The html tag to wrap the title text in. Will default to a `div`                                       |
+| type       | string  | false    | The type of panel to render. Available options are `success`, `warn`, `error` and `branded`           |
+| spacious   | boolean | false    | Will render a more spacious version of the panel if set to `true`                                     |
+| classes    | string  | false    | Custom classes to add to the panel                                                                    |
+| id         | string  | false    | Custom id to add to the panel                                                                         |
+| attributes | object  | false    | HTML attributes to apply to the panel (e.g. data attributes)                                          |
+| icon       | string  | false    | Set this to the name of the icon you want to be included before the contents of the panel             |
+| iconSize   | string  | false    | Set this to the size of the icon you want can be set to "m", "l" "xl". If not set will defualt to "m" |
```

Comment:
Think the default needs to be the smallest icon (to match height of base font size), then the settings m, l, and xl match the three next upper font sizes

Reply:
I've pushed up a temporary success panel examples as there's still alignment and size issues.

<img width="824" alt="Screenshot 2020-09-18 at 10 37 36" src="https://user-images.githubusercontent.com/43346934/93583033-32750b80-f99b-11ea-939c-36d2101c4173.png">

Reply:
Ive now sorted this and added spacing between the text and icon for larger icon sizes